include/oneapi/dpl/pstl/onedpl_config.h

```diff
@@ -181,13 +183,21 @@
 #    define _ONEDPL_PRAGMA_SIMD_ORDERED_MONOTONIC_2ARGS(PRM1, PRM2)
 #endif
 
-#if (__INTEL_COMPILER >= 1900 || !defined(__INTEL_COMPILER) && _ONEDPL_GCC_VERSION >= 40900 || _OPENMP >= 201307)
+#if (_OPENMP >= 201307 || __INTEL_LLVM_COMPILER || __INTEL_COMPILER >= 1900 ||                                         \
+     !defined(__INTEL_LLVM_COMPILER) && !defined(__INTEL_COMPILER) && _ONEDPL_GCC_VERSION >= 40900)
 #    define _ONEDPL_UDR_PRESENT 1
 #else
 #    define _ONEDPL_UDR_PRESENT 0
 #endif
 
-#define _ONEDPL_UDS_PRESENT (__INTEL_COMPILER >= 1900 && __INTEL_COMPILER_BUILD_DATE >= 20180626)
+// TODO: enable with Intel LLVM-based compiler on Windows when fixed
+#if __INTEL_LLVM_COMPILER >= 20230100 && !defined(_MSC_VER)
+#    define _ONEDPL_UDS_PRESENT 1
+#elif __INTEL_COMPILER >= 1900 && __INTEL_COMPILER_BUILD_DATE >= 20180626
```

Comment:
Probably, it makes sense to write one "common" condition instead of two(194 and 196 lines) and keep just one `#define _ONEDPL_UDS_PRESENT 1`?

Reply:
It's been changed to have only one `#define _ONEDPL_UDS_PRESENT 1`.